test/parallel/test-repl-function-definition-edge-case.js

```diff
@@ -9,7 +9,8 @@ const r = initRepl();
 
 r.input.emit('data', 'function a() { return 42; } (1)\n');
 r.input.emit('data', 'a\n');
-r.input.emit('data', '.exit');
+r.input.emit('data', '.exit\n');
+r.once('exit', common.mustCall());
```

Comment:
this new `mustCall` assertion is unnecessary, I can remove it if people don't like this addition

Note: prior to the `\n` addition this would cause the test to fail